spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe "movie show page", type: :feature do
+  it "shows movie title, creation year, and genre" do
+    studio1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    movie1 = studio1.movies.create!(title: "Raiders of the Lost Ark", creation_year: 1981, genre: "Action/Adventure")
+    
+    visit "/movies/#{movie1.id}"
+    
+    expect(page).to have_content("Raiders of the Lost Ark")
```

Comment:
It's best to anchor this data to labels that describe what it refers to, and then test for the whole text. For example, expecting the page to have "Raiders of the Lost Ark, Released in 1981" is more robust than expecting just Raiders and 1981 as separate assertions. That text could be anywhere so it's more powerful to anchor these fields together. Even more robust would be to add within blocks to assert that this data appears in the section for Bros and not elsewhere